app/src/main/java/com/jerboa/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -224,6 +227,15 @@ fun LookAndFeelActivity(
                     },
                     onCheckedChange = { updateAppSettings() },
                 )
+                AnimatedVisibility(visible = showBottomNavState.value) {
```

Comment:
I'm not sure how I feel about making a setting visible based on another setting. I think this should just be always shown.

Reply:
Are you sure? This setting has no effect when the navbar is disabled. So it makes sense to hide it imo. If you insist, I will remove that.

Reply:
It has no effect, but I am concerned that hiding it would be confusing and frustrating to users who are trying to find that specific option. It would make more sense perhaps to disable/gray out the option so that users can at least see that it would be there than hiding it entirely.

Reply:
I guess that's a compromise.